examples/all-clusters-app/ameba/main/chipinterface.cpp

```diff
@@ -142,6 +142,23 @@ static void InitServer(intptr_t context)
 #endif
 }
 
+extern "C" bool matter_server_is_commissioned()
+{
+    return (chip::Server::GetInstance().GetFabricTable().FabricCount() != 0);
```

Comment:
This is going to return "true" after AddNOC during commissioning, when the device is not really "commissioned" yet (has no network info, fail-safe can roll everything back, etc).

How should that case be handled here?

Reply:
Indeed, thank you for the suggestion. It is true that checking the fabricCount alone is not sufficient to draw the conclusion of 'commissioned'. We will make modifications.